alphagov/paas-admin
# Introduce the use of /v2/info

## What

Currently, we have to provide a lot of environment variables before
running the application. This is annoying some of these could be sliced
out.

We're going to request `/v2/info` and set those up in the application
itself, to reduce devops stuff.

## How to review

- Make sure the tests pass
- Run the application
- Make sure you can log in
- You can check if the log on start shows correct urls.

## Who can review

Neither @chrisfarms nor myself.
